google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineAssetProvider.java

```diff
@@ -44,4 +48,20 @@ public XmlFile loadAppEngineStandardWebXml(@NotNull Project project, @NotNull Ar
 
     return null;
   }
+
+  @Nullable
+  @Override
+  public XmlFile loadAppEngineStandardWebXml(@NotNull Project project, @NotNull Module module) {
+    Collection<VirtualFile> appEngineWebXmls
+        = FilenameIndex.getVirtualFilesByName(
+            project, "appengine-web.xml", module.getModuleContentScope());
+
+    for (VirtualFile contentRoot : appEngineWebXmls) {
```

Comment:
why is the virtual file called contentRoot?

Also I'm a bit worried if the user for some reason has multiple appengine-web.xml files in their src roots that this is going to behave erratically.  Could we maybe log a warning if there are multiple?  Also potentially prefer appengine-web.xmls that have WEB-INF as a parent.